pages/design-principles/design-principles.md

```diff
@@ -273,7 +273,7 @@ subnav:
 #### Government resources
 
 - [Guide to the Digital Analytics Program (DAP)](https://digital.gov/services/dap/) [digital.gov]
-- [Getting started with Search.gov](https://www.youtube.com/watch?v=p-y9T23ziEg) [search.gov]
+- [Getting started with Search.gov[youtube.com]](https://www.youtube.com/watch?v=p-y9T23ziEg) [search.gov]
```

Comment:
Needs a space before `[youtube.com]`. Do we need the additional `[search.gov]` at the end of the line?

I see this:
![image](https://user-images.githubusercontent.com/3385219/140785491-53f0249e-5a8c-479a-9f71-678e7ef95365.png)

[Preview →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/brunerae-innovation-search-toptasks/design-principles/)